shell/src/test/java/tachyon/shell/TfsShellTest.java

```diff
@@ -611,10 +628,10 @@ public void renameToExistingFileTest() throws IOException {
 
     StringBuilder toCompare = new StringBuilder();
     mFsShell.run("mkdir", "/testFolder");
-    toCompare.append(getCommandOutput(new String[] {"mkdir", "/testFolder"}));
+    toCompare.append(getCommandOutput(new String[]{"mkdir", "/testFolder"}));
     mFsShell.run("mkdir", "/testFolder1");
-    toCompare.append(getCommandOutput(new String[] {"mkdir", "/testFolder1"}));
-    mFsShell.rename(new String[] {"rename", "/testFolder1", "/testFolder"});
+    toCompare.append(getCommandOutput(new String[]{"mkdir", "/testFolder1"}));
+    mFsShell.rename(new String[]{"rename", "/testFolder1", "/testFolder"});
```

Comment:
Was this suggested by the auto formatter? It seems like the common style is to have a space in the codebase.